scripts/known_good.json

```diff
@@ -17,7 +17,7 @@
       "sub_dir" : "Vulkan-Headers",
       "build_dir" : "Vulkan-Headers/build",
       "install_dir" : "Vulkan-Headers/build/install",
-      "commit" : "v1.1.119"
+      "commit" : "9eab96f40634c81ed53d54d2c9df126a445dfd56"
```

Comment:
Why is this a commit ID and not a tag?

Reply:
Because the tag points to a commit that produces build errors when generating source scripts in all the repos and the SHA indicates the commit that fixes it.

Reply:
Sounds good!

Reply:
Ideally there should be a new tag, indicating the best commit to grab for the headers (and their related resources; I realize the headers themselves weren't changed)

Reply:
Indicating the best commits to use for building a repo is the purpose of the known-good file.

Reply:
Yes, and the other person who experienced the error.
Tags are used to indicate "immutable commits of interest".
I'd assert that every reference in known-good files, by definition, should be a tag.